### DISCUSSION TITLE
Get SessionId to send it to own backend

### DISCUSSION THREAD

**Discussion by asurovenko-zeta** [nextauthjs/next-auth#4458] [Help]
### Question 💬

I need to provide sessionId to my own backend to validate access to endpoint. Can't find it in google so far

### How to reproduce ☕️

```const session = useSession();```
I already have session object, but there's no sessionId value from database.

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by ThangHuuVu** [comment]
Did you try using the [session callback](https://next-auth.js.org/configuration/callbacks#session-callback)? 
You can assign the `sessionId` to the session object
```js
callbacks: {
  async session({ session, token, user }) {
    session.sessionId = sessionId
    return session
  }
}
```

**Reply by jimmailcamp** [reply]
Is thisa joke?
